src/e2e-test/java/io/cdap/plugin/BQValidation.java

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
```

Comment:
Please change the year from 2022 to 2023.

Reply:
done